src/main/kotlin/no/nav/tiltaksarrangor/ingest/IngestService.kt

```diff
@@ -260,6 +281,45 @@ class IngestService(
 	}
 }
 
+private fun DeltakerDbo.hentPersonaliaOppdateringer(nyDeltaker: DeltakerDto): Oppdatering.NavBrukerEndring? {
+	val telefonnummer = if (this.telefonnummer ==
```

Comment:
Her ville jeg kanskje brukt bool isteden for å si `bool telefonEndret` så er det litt lettere å forstå lengre ned at man skal returnere null gitt at tlf og epost ikke er endret